target/bin/addsaslpassword

```diff
@@ -2,34 +2,29 @@
 
 DATABASE=${DATABASE:-/tmp/docker-mailserver/postfix-sasl-password.cf}
 
-DOMAIN="$1"
-USER="$2"
-PASSWD="$3"
-
-usage() {
-	echo "Usage: addsaslpassword <domain> <username> <password>"
-}
-
-errex() {
-	echo "$@" 1>&2
-	exit 1
-}
-
-escape() {
-	echo "${1//./\\.}"
-}
-
-[ -z "$DOMAIN" ] && { usage; errex "no domain specified"; }
-[ -z "$USER" ] && { usage; errex "no username specified"; }
-
-if [ -z "$PASSWD" ]; then
-	read -s -p "Enter Password: " PASSWD
-	echo
-	[ -z "$PASSWD" ] && errex "Password must not be empty"
+DOMAIN="${1}"
+USER="${2}"
+PASSWD="${3}"
+
+function usage  { echo "Usage: addsaslpassword <domain> <username> <password>" ; }
```

Comment:
This part repeats itself in the other scripts, would be a candidate to move it into a shared file and source it from there.

Reply:
I have outsourced what I thought appropriate into a new file and sourced this file from every other file which needs the mentioned functions. This is done in the latest commit.